server/models/itineraryModel.js

```diff
@@ -1,22 +1,22 @@
 const mongoose = require('mongoose')
 
-const itineraryStageSchema = new mongoose.Schema({
+const stageSchema = new mongoose.Schema({
   stage: Number,
   place: String,
   description: String,
   address: String,
   cost: Number,
 })
 
-const itinerarySchema = new mongoose.Schema({
+const tripSchema = new mongoose.Schema({
   destination: String,
   budget: Number,
   days: Number,
   stages: Number,
   preferences: [String],
-  plan: [itineraryStageSchema],
+  plan: [stageSchema],
 })
 
-const Itinerary = mongoose.model('Itinerary', itinerarySchema)
+const Itinerary = mongoose.model('trip', tripSchema)
```

Comment:
should we rename this to const Trip? or is that too generic?

Reply:
This file shouldn't exist. I'm going to comment it out actually